apps/api/v2/src/modules/auth/guards/organizations/is-admin-access.guard.ts

```diff
@@ -0,0 +1,61 @@
+import { OrganizationsRepository } from "@/modules/organizations/organizations.repository";
+import { RedisService } from "@/modules/redis/redis.service";
+import { Injectable, CanActivate, ExecutionContext, ForbiddenException } from "@nestjs/common";
+import { Request } from "express";
+
+import { Team } from "@calcom/prisma/client";
+
+type CachedData = {
+  org?: Team;
+  canAccess?: boolean;
+};
+
+@Injectable()
+export class IsAdminAccessGuard implements CanActivate {
```

Comment:
I would rename the guard to `hasAdminAccessGuard`

Reply:
I think `IsAdminAPIEnabled` actually fits better, and aligns with the DB change as well. I'll proceed with that 🙏